core_v2/sources/domains.move

```diff
@@ -692,34 +690,24 @@ module aptos_names_v2::domains {
     /// Entry function for clearing reverse lookup.
     public entry fun clear_reverse_lookup_entry(
         account: &signer
-    ) acquires CollectionCapabilityV2, NameRecordV2, ReverseRecord, SetReverseLookupEventsV1 {
+    ) acquires NameRecordV2, ReverseRecord, SetReverseLookupEventsV1 {
         clear_reverse_lookup(account);
     }
 
     /// Clears the user's reverse lookup.
     public fun clear_reverse_lookup(
         account: &signer
-    ) acquires CollectionCapabilityV2, NameRecordV2, ReverseRecord, SetReverseLookupEventsV1 {
+    ) acquires NameRecordV2, ReverseRecord, SetReverseLookupEventsV1 {
         let account_addr = signer::address_of(account);
         clear_reverse_lookup_internal(account_addr);
     }
 
-    inline fun reverse_record_addr(account_addr: address): address acquires CollectionCapabilityV2 {
-        // TODO: Use object ExtendRef
-        object::create_object_address(
-            &get_token_signer_address(),
-            bcs::to_bytes(&account_addr),
-
-        )
-    }
-
     /// Returns the reverse lookup (the token addr) for an address if any.
     public fun get_reverse_lookup(
         account_addr: address
-    ): Option<address> acquires CollectionCapabilityV2, ReverseRecord {
-        let reverse_record_addr = reverse_record_addr(account_addr);
-        if (is_object(reverse_record_addr)) {
-            let reverse_record = borrow_global<ReverseRecord>(reverse_record_addr);
+    ): Option<address> acquires ReverseRecord {
+        if (exists<ReverseRecord>(account_addr)) {
+            let reverse_record = borrow_global<ReverseRecord>(account_addr);
```

Comment:
does this mean there would always be only 1 `ReverseRecord` resource under `account_addr`?